openeogeotrellis/logs.py

```diff
@@ -101,6 +123,11 @@ def _elasticsearch_logs(
                 }
             }
         )
+    if level_filter:
+        query["bool"]["filter"].append(level_filter)
+        print(f"{level_filter=}")
```

Comment:
remove the print statements (or replace them with standard `logging.debug` calls)

Reply:
Removed them.